app/views/names/map.html.erb

```diff
@@ -3,7 +3,7 @@
 
   tabs = [
     link_to(:name_map_about.t(name: @name.display_name),
-            name_path(@name.show_link_args, q: get_query_param),
+            name_path(@name, q: get_query_param),
```

Comment:
Original appears to be a typo. I changed it to what I think was intended.

Reply:
I believe you're right!